training-react/src/app/components/LoadingScreen/index.js

```diff
@@ -1,14 +1,16 @@
 import React, { Component } from 'react';
 
 import LoadingItem from '../LoadingItem';
+import UserDetails from '../User/layout';
 
-const withLoadingScreen = WrappedComponent =>
-  class LoadingScreen extends Component {
+function withLoadingScreen(WrappedComponent) {
+  return class LoadingScreen extends Component {
     render() {
       const { isLoading } = this.props;
       if (isLoading) return <LoadingItem />;
       return <WrappedComponent {...this.props} />;
     }
   };
+}
 
-export default withLoadingScreen;
+export default withLoadingScreen(UserDetails);
```

Comment:
This was ok before, this `withLoadingScreen(UserDetails)` should be done in `UserDetails` definition file